src/DataFixtures/PageFixtures.php

```diff
@@ -8,8 +8,8 @@
 use App\Service\FileUploaderInterface;
 use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Common\Persistence\ObjectManager;
-use Symfony\Component\HttpFoundation\File\File;
 use Faker\Factory as Faker;
+use Symfony\Component\HttpFoundation\File\File;
```

Comment:
This should not be in this PR, could you make another PR for that? It has nothing to do with the current feature.